### ISSUE TITLE
Separate chats are left in LHN after completing task until login/logout

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#19149]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Login to NewDot
2. Create Task
3. Assign the Task to another user
4. Mark as Done
5. Repeat steps 2 and 3 several times
6. Observe task chats are displayed in the bottom of LHN
7. Logout
8. Login
9. Observe they disappear 

## Expected Result:
I assume that Task chats should not be created after step 2 like it is displayed on PR demo video.

If Task/chat creation is valid then they should disappear from LHN after completed and only show in the chat thread.
If Tasks should stay on LHN, then they should not disappear after logout / login

## Actual Result:

Chat appear in LHN after task creation. After marking it completed, they move down in LHN. 
After logout/login chat disappear from LHN and can only be found when going to the chat with the user. 

![image](https://github.com/Expensify/App/assets/51066321/e0507c4d-b18d-40e4-a482-8eef9bdbaeee)

## Workaround:
unknown 

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [X] MacOS / Chrome / Safari
- [X] MacOS / Desktop

**Version Number:** v1.3.15-5
**Reproducible in staging?:** Y 
**Reproducible in production?:** N 
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
![Screen Shot 2023-05-17 at 6 39 43 PM](https://github.com/Expensify/App/assets/43995119/ada3b869-a908-43a3-8fec-88a00fd41cf7)

https://github.com/Expensify/App/assets/43995119/8a2d5f26-57d7-47ea-b501-bb55a7f868d0

Found When executing PR #18806

**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team 
**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by isabelastisser** [comment]
OOO reassigning!

**Comment by Christinadobrzyn** [comment]
I can reproduce this and I do think something odd is happening - either tasks need to not show once completed in the LHN or they need to always show. 

Found related GH - https://github.com/Expensify/App/issues/18510 

cc @thienlnam do you think this is something related to the ^ job?

**Comment by thienlnam** [comment]
Yeah this is expected - we should be showing tasks in the LHN now! We have another issue to fix fetching the tasks in the LHN though
